packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -462,26 +493,148 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Some popups, like iOS's volume control popup, choose to render a blurred
   /// area without any white paint covering it. To achieve this effect,
   /// [isSurfacePainted] should be set to false.
+  ///
+  /// Defaults to true.
   final bool isSurfacePainted;
 
   /// The widget below this widget in the tree.
-  final Widget? child;
+  ///
+  /// Because [CupertinoPopupSurface] is composed of proxy boxes, which mimic
+  /// the size of their child, a [child] is required to ensure that this surface
+  /// has a size.
+  final Widget child;
+
+  /// The default strength of the blur applied to widgets underlying a
+  /// [CupertinoPopupSurface].
+  ///
+  /// Setting this to 0 will remove the blur filter.
+  static const double defaultBlurSigma = 30.0;
+
+  /// The default corner radius of a [CupertinoPopupSurface].
+  static const BorderRadius _clipper = BorderRadius.all(Radius.circular(_kCornerRadius));
+
+  /// The [ColorFilter.matrix] used to saturate widgets underlying a
```

Comment:
Can you make all docs for private variables use `//` instead of `///`? Usually it's not a big deal, but the explanations on how the matrices are derived really should use `//`.

Reply:
Done! I removed the interpolation brackets and code backticks as well.